### DISCUSSION TITLE
Creating list with Enter key still shows modal in progress

### DISCUSSION THREAD

**Discussion by gregkonush** [community/community#81925] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The list is still created however after refreshing the entire browser window

![image](https://github.com/community/community/assets/12027037/53603256-da63-4062-abfc-50dc334c63d4)

**Comment by gregkonush** [comment]
Bonus bug, highlighting the report bug link with tab and key and pressing enter afterwards redirects to the bug reporting page but the modal is still open.

**Comment by Jamim** [comment]
This discussion duplicates #50880, as a dozen of other reports.
